game/script.rpy

```diff
@@ -922,10 +925,212 @@ label conservatory_menu:
+
+  "I didn't know if this version of Poseidon's curse included some heightened reflection, some self-consciousness that brought her to her knees more than any sort of bestial lust could."
+
+  "In the end, Pasiphae carried out her plan against the backdrop of an uncaring world."
+
+  "The consequences of her actions showed her unraveling, the play ending ultimately as she held up her bloodied child, the minotaur, her face contorting with laughter as she died."
+
+  if pasiphae_author == "Marian":
+
+    "I thought about the implications of Marian having written this."
+
+  jump pasiphae_author_menu
+
+else:
+
+  "There was a play of some sort in the book. I didn't quite understand it, but it did seem to refer to some Greek myth."
+
+  "They certainly were a strange lot, back then."
+
+  "Strange, but fascinating."
+
+  "As I leafed through the pages, I eventually got to a description of a wooden bull, and some rather... wild events."
+
+  "{i}{b}WOODEN BULL{/b} enters right. It is made of heavy mahogany, with wheels of iron.{/i}"
+
+  "{i}Inlaid are sapphires around its neck, rubies on its hooves, a pearl on its forehead shaped like a crescent moon.{/i}"
+
+  "{i}Its horns are proud, comely, its udders hang heavily. Its tail is goldenrod braided silk. Clearly visible on its side is a hinged entrance. Within is Pasiphae.{/i}"
+
+  "{b}Pasiphae (booming, within the wooden bull){/b}: Bring him! Bring the mighty bull! I can stand it no longer! Thus is the decree of your queen! See me! The heat that burns within me cannot be quenched by mere man."
+
+  "(beat)"
+
+  "{b}Chorus{/b}: Lust, a curse, only ruination it brings. The weak, the strong, all tremble beneath it. The powerful dive in, and they leave a ripple, deeper in, the stronger they are. Merciless Aphrodite reels her line tightly."
+
+  "{b}Pasiphae{/b}: Can you not see? Aphrodite holds no sway over me. Hear me. The sound of my voice is my own. You look at me and see a daughter of Helios. Is that not proof enough?"
+
+  "{b}Chorus{/b}: She says she is under no curse. Surely she can control herself, our mighty queen, sorceress, wise ruler. There is no sense in what she does. Where is the truth in her words? Is she mad or is she cursed?"
+
+  "{b}Pasiphae{/b}: Do you not heed your queen? Not once, but twice you disobey my call. Behold me. I am she. There is no curse here. Bring forth the bull that I seek. You have allowed me to come this far, would you try and stop me now?"
+
+  "{i}THE KING'S BULL{/i} enters left and approaches {i}WOODEN BULL{/i}"
+
+  "{b}Pasiphae{/b}: Alight! Yes! This is sweet madness. Witness me!"
+
+  "There was then a very graphic description about the real bull mounting the wooden bull on the stage and Pasiphae crying out in 'ecstatic pain.'"
+
+  "Part of me was downright leery of the mind that recreated this scene. The other part of me wondered where the air holes would be in this wooden bull."
```

Comment:
Ok, I have to admit I was skeptical of choosing this myth but I think you've convinced me! If I can get over the bestiality part. Let's discuss!